packages/content/package.json

```diff
@@ -26,7 +26,7 @@
   },
   "homepage": "https://github.com/bluecadet/launchpad/packages/content",
   "dependencies": {
-    "@bluecadet/launchpad-utils": "*"
+    "@bluecadet/launchpad-utils": "~1.0.1"
```

Comment:
@claytercek do you know if we have to specify the version here or can changeset handle that?

Reply:
Ok wonderful. So let's keep specific versions, esp if changeset updates those